src/accordian/Accordion.tsx

```diff
@@ -160,7 +169,7 @@ export class Accordion extends React.Component<AccordionProps, AccordionState> {
                                 type="button"
                                 aria-disabled="false"
                                 aria-controls="clr-accordion-content"
-                                aria-expanded="false"
+                                aria-expanded={index === prevItemIndex || (accordionMultiPanel && content.isOpen)}
```

Comment:
Please create small function for this condition which will take index as parameter as it is repeating many palaces.